features/SaaSquatch-to-Hubspot/Sync Participants with HubSpot.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Sync Participants with HubSpot
+
+    As a user, I want to be able to sync participants in SaaSquatch as contacts in HubSpot
+
+    Background:
+        Given the integration is active
+
+    Scenario: Import existing participants in SaaSquatch as contacts in HubSpot
+        Given a participant exists in SaaSquatch
+        When I have toggled the import existing participants option on and selected the fields to sync
+        Then a contact is created in HubSpot with those fields
+
+    Scenario: Do not import existing participants in SaaSquatch as contacts in HubSpot
+        Given a participant exists in SaaSquatch
+        When I have toggled the import existing participants option off
+        Then contacts are not created in HubSpot
+
+    Scenario: Create contact in HubSpot when a participant is created in SaaSquatch
+        Given a participant is created in SaaSquatch
+        When I have toggled the create new contact option on and selected the fields to sync
+        Then a new contact is created in HubSpot with those fields
+
+    Scenario: Do not create contact in HubSpot when a participant is created in SaaSquatch
+        Given a participant is created in SaaSquatch
+        When I have toggled the create new contact option off
+        Then a new contact is not created in HubSpot
+
+    Scenario: Delete contact in HubSpot when the participant is deleted in SaaSquatch
+        Given a participant is deleted in SaaSquatch
+        When I have toggled the delete contact option on
+        Then the contact is deleted in HubSpot
+
+    Scenario: Do not delete contact in HubSpot when the participant is deleted in SaaSquatch
+        Given a participant is deleted in SaaSquatch
+        When I have toggled the delete contact option off
+        Then the contact is not deleted in HubSpot
+
+    Scenario: Import existing referral links to contact in HubSpot from participant in SaaSquatch
+        Given a contact exists in HubSpot and the particpant exists in SaaSquatch
+        When I have toggled the import existing referral links option on and selected what referral programs to sync
```

Comment:
@EvanDungate do we know if we can check what referral programs a participant belongs to in SaaSquatch easily from the API?